### DISCUSSION TITLE
Problem with 4.2.7 and ABL

### DISCUSSION THREAD

**Discussion by oricardomiranda** [Jyers/Marlin#1281] [Q&A] [answered]
Hello

I have an Ender 3 V2 with a 4.2.7 board.

I have an ABL so I've been trying to use the open source folder but without any luck.

The axels just won't move. Like when we choose the wrong motherboard file. 
[Marlin-1.3.5 to upload.zip](https://github.com/Jyers/Marlin/files/6949783/Marlin-1.3.5.to.upload.zip)

I've been successful with other firmwares before.

I think it might be my board, because in the other firmwares I get to choose my board. Is this source I can't seem to find it. Can someone help me?

If I use a configuration from the list it works fine. But I as there's none for abl, I really need to compile my own. 

I'm attaching a zip file with my configs

**Comment by Jyers** [comment, marked as answer]
Did you end up resolving this?

**Reply by oricardomiranda** [reply]
Yes I did. I got a hint and 10 minutes later it was solved. Thank you so much for this firmware mr jyers
